playbook-website/app/javascript/site_styles/_visual_guidelines.scss

```diff
@@ -41,4 +44,61 @@
     .nine { margin-left: 400px; }
     .ten { margin-left: 450px; }
   }
-}
\ No newline at end of file
+
+
+  #border_radius_container{
+    text-align: center;
+  }
+
+  #border_radius_rounded{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_rounded;
+  }
+
+  #border_radius_xl{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_xl;
+  }
+
+  #border_radius_lg{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_lg;
+  }
+
+  #border_radius_md{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_md;
+  }
+
+  #border_radius_sm{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_sm;
+  }
+
+  #border_radius_xs{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    margin: 20px 15px 15px;
+    border-radius: $border_radius_xs;
+  }
+
+  #border_radius_none{
+    border: $border_light 1px solid;
+    height: 100px;
+    width: 100px;
+    border-radius: $border_radius_none;
+  }
```

Comment:
In order to keep this file clean lest move this into a partial and import it here.

Also you need a space after your selector and your curly brace `.selector {`